src/include/daos_mgmt.h

```diff
@@ -323,6 +323,24 @@ daos_pool_remove_replicas(const uuid_t uuid, const char *group,
 			  d_rank_list_t *svc, d_rank_list_t *targets,
 			  d_rank_list_t *failed, daos_event_t *ev);
 
+/**
+ * List all pools created in the specified DAOS system.
+ *
+ * \param group	[IN]		Name of DAOS system managing the service.
+ * \param pools	[OUT]		Array of pool mgmt information structures.
+ *				NULL is permitted in which case only the
+ *				number of pools will be returned in \a npools.
```

Comment:
Are there any memory objects that callers need to free (with D_FREE or something else?) if rc == 0?

Reply:
Yes, updating documentation to indicate caller (on successful return) must free the dynamically-allocated service replica rank lists (mgpi_svc) in each item of the pools array. With d_rank_list_free().